etna/packages/magby/magby/Magby.py

```diff
@@ -7,14 +7,16 @@
 _session = Session()
 
 class Magby(object):
-    def __init__(self, url: str, token: str):
+    def __init__(self, url: str, token: str, verify: bool = True):
```

Comment:
@Gvaihir unfortunately I had to edit the Magby constructor slightly to get the UMAP Vulcan work functional in dev. This should go away once we clean up the docker networking issues, but that will require a bit of work on the Magma app.